fvm/env.go

```diff
@@ -612,6 +619,51 @@ func (e *hostEnv) RemoveAccountKey(address runtime.Address, index int) (publicKe
 	return e.transactionEnv.RemoveAccountKey(address, index)
 }
 
+func (e *hostEnv) AddAccountKey(
```

Comment:
Ideally these logic should be moved into an accountKey handler and all unit tests related to this functionality happens there. and here we just redirect the calls to a account key manager, similar to the way we handle contracts.

Reply:
Great idea!

Could we do that in a separate PR, so this one can get sipped and also doesn't get too large?

Reply:
yeah, sounds like a good idea, I can take care of that, I have to break several parts into handlers so I can move this as well.